ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -839,113 +843,168 @@ private Material GenerateGLTFMaterialFromRenderMaterial(VMaterial renderMaterial
             }
 
             material.WithPBRMetallicRoughness(baseColor, null, metallicFactor: metalValue);
+            using var ormTexture = new TextureExtract.TexturePacker { DefaultColor = new SkiaSharp.SKColor(255, 255, 0, 255) };
+            var ormHasAO = false;
```

Comment:
ormHasAmbientOcclusion

And please add a comment above saying orm is Occlusion, Roughness, Metallic.

Reply:
Should be more clearer now.